protostar/testing/cheatcodes/expect_call_cheatcode.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING, Callable
+
+from starkware.starknet.public.abi import get_selector_from_name
+
+from protostar.starknet.cheatcode import Cheatcode
+from protostar.testing.hook import Hook
+from protostar.testing.test_environment_exceptions import ExpectedCallException
+
+
+if TYPE_CHECKING:
+    from protostar.starknet.forkable_starknet import ForkableStarknet
+
+
+class ExpectCallCheatcode(Cheatcode):
+    def __init__(
+        self,
+        syscall_dependencies: Cheatcode.SyscallDependencies,
+        starknet: "ForkableStarknet",
+        finish_hook: Hook,
+    ):
+        super().__init__(syscall_dependencies)
+        self.starknet = starknet
```

Comment:
you don't need this?

Reply:
right, thx, [remove unnecessary arg starknet](https://github.com/software-mansion/protostar/pull/1060/commits/755fceeb71c7e2b0a7c4e9ca0443a03bfdb07474)